perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -660,6 +660,48 @@ sub body {
     return $body;
 }
 
+=head2 body_object
+
+Similar to C<body> but returns a body object, either from C<external_body>
+or from C<bodies_str>.
+
+NB: For cobrands where C<bodies_str> is a comma-separated list, this may
+return the first retrieved body at random.  (This behaviour is currently
+intended for Zurich, which uses a single body.)
+
+=cut
+
+sub body_object {
```

Comment:
This doesn't ever seem to be used anywhere, and should probably have been removed when I updated the template to use `problem->response_templates`.